products/sle/main.pm

```diff
@@ -1022,37 +1022,14 @@ else {
         load_reboot_tests();
         loadtest "migration/post_upgrade";
     }
-    elsif (get_var("BOOT_HDD_IMAGE") && !is_jeos) {
+    elsif (get_var('BOOT_HDD_IMAGE') && get_var('RT_TESTS') && !is_jeos) {
```

Comment:
You are changing logic for following `elsif` statements, did you check that it won't break rt tests?